packages/config/src/cookieTransform.js

```diff
@@ -1,3 +0,0 @@
-const cookieTransform = require('@redhat-cloud-services/frontend-components-config-utilities/cookieTransform');
```

Comment:
We are almost good to merge. But re-export also has to stay for now. Can you add a deprecated console warning to import these directly from the `@redhat-cloud-services/frontend-components-config-utilities` package? These re-exports are leftover before the config split happened.